src/features/createSurvey/addSurvey/choiceForm/Check.tsx

```diff
@@ -0,0 +1,62 @@
+import { css, type Theme } from '@emotion/react';
+
+import { SmallCheckIcon } from '~/components/icons/CheckIcon';
+import { BODY_2_BOLD } from '~/styles/typo';
+
+interface Props {
+  isChecked: boolean;
+  toggleCheck: () => void;
+  label: string;
+}
+
+const Check = ({ isChecked, toggleCheck, label }: Props) => {
+  return (
+    <div css={containerCss}>
+      <button id="multi-check" type="button" css={(theme) => checkBoxCss(theme, isChecked)} onClick={toggleCheck}>
+        {isChecked && <SmallCheckIcon />}
+      </button>
+      <label htmlFor="multi-check" css={checkboxLabelCss}>
+        {label}
+      </label>
```

Comment:
접근성 👍 👍 👍 

```suggestion
const ID = 'multi-check';

      <button id={ID} type="button" css={(theme) => checkBoxCss(theme, isChecked)} onClick={toggleCheck}>
        {isChecked && <SmallCheckIcon />}
      </button>
      <label htmlFor={ID} css={checkboxLabelCss}>
        {label}
      </label>
```

상수화 해두면 어떨까요 ?!

Reply:
좋습니다! 
근데 상수화를 통해 얻을 수 있는 어떤 장점이 있는지 궁금합니다! @hyesungoh

Reply:
일반적인 상수화라면 재사용성, 유지보수, 휴먼 에러 방지, 가독성 ... 정도가 있을 거 같은데요

지금 상황에서는 휴먼 에러 + 가독성 정도가 해당되는 거 같아요

아니면 useId를 사용해도 괜찮다고 생각되네요 !
https://react.dev/reference/react/useId @sumi-0011